docs/usage.rst

```diff
@@ -195,3 +195,5 @@ This is the case when you choose to integrate your user registration process wit
             return self.serializer_class
 
 Otherwise, serializer classes for ``verify`` and ``register`` views will not be available.
+
+3. Latest passcode generated for a number can also be found at `/admin/phone_verify/smsverification/`.
```

Comment:
```suggestion
3. Latest passcode generated for a ``phone_number`` can also be found at `/admin/phone_verify/smsverification/` URL.
```

Reply:
Can you please also unify the term we use in the documentation viz (security code, passcode, OTP) to security code, please?

Reply:
The request and responses are using `otp` as a parameter. It is also being used in many other places. Would it be better to unify the terms to `OTP` instead of `security code`?

Reply:
Ah, we need to also have the functionality of OTP here. It should've been actually called `security_code` but I'm not sure if I'd like to break the APIs.